presto-main/src/test/java/com/facebook/presto/sql/planner/TestRowExpressionFormatter.java

```diff
@@ -116,11 +143,74 @@ public void testConstants()
         assertEquals(format(constantExpression), "[Block: position count: 2; size: 96 bytes]");
     }
 
+    @Test
+    public void testCalls()
+    {
+        // setup
+        RowExpression callExpression;
+
+        // arithmetic
+        callExpression = exampleCallExpression(ADD);
+        assertEquals(format(callExpression), "(#1) + (BIGINT 5)");
+        callExpression = exampleCallExpression(SUBTRACT);
+        assertEquals(format(callExpression), "(#1) - (BIGINT 5)");
+        callExpression = exampleCallExpression(MULTIPLY);
+        assertEquals(format(callExpression), "(#1) * (BIGINT 5)");
+        callExpression = exampleCallExpression(DIVIDE);
+        assertEquals(format(callExpression), "(#1) / (BIGINT 5)");
+        callExpression = exampleCallExpression(MODULUS);
+        assertEquals(format(callExpression), "(#1) % (BIGINT 5)");
+
+        // comparison
+        callExpression = exampleCallExpression(GREATER_THAN);
+        assertEquals(format(callExpression), "(#1) > (BIGINT 5)");
+        callExpression = exampleCallExpression(LESS_THAN);
+        assertEquals(format(callExpression), "(#1) < (BIGINT 5)");
+        callExpression = exampleCallExpression(GREATER_THAN_OR_EQUAL);
+        assertEquals(format(callExpression), "(#1) >= (BIGINT 5)");
+        callExpression = exampleCallExpression(LESS_THAN_OR_EQUAL);
+        assertEquals(format(callExpression), "(#1) <= (BIGINT 5)");
+        callExpression = exampleCallExpression(EQUAL);
+        assertEquals(format(callExpression), "(#1) = (BIGINT 5)");
+        callExpression = exampleCallExpression(NOT_EQUAL);
+        assertEquals(format(callExpression), "(#1) <> (BIGINT 5)");
+        callExpression = exampleCallExpression(IS_DISTINCT_FROM);
+        assertEquals(format(callExpression), "(#1) IS DISTINCT FROM (BIGINT 5)");
+
+        // nested expressions
+        RowExpression temp = exampleCallExpression(ADD);
```

Comment:
Just call it `expression`. We usually spell out a word completely than using abbreviations.